inputfiles/idlSources.json

```diff
@@ -108,6 +108,10 @@
         "url": "https://www.w3.org/TR/css-overscroll-1/",
         "title": "CSS Overscroll Behavior"
     },
+    {
+        "url": "https://www.w3.org/TR/css-pseudo-4/",
+        "title": "CSS Pseudo-Elements"
```

Comment:
This is [being used by the Web Animation specification](https://www.w3.org/TR/web-animations-1/#extensions-to-the-pseudoelement-interface), so we need to include its types. Fortunately, [there aren't that many new types](https://www.w3.org/TR/css-pseudo-4/#idl-index).

Reply:
Neither Chrome nor Firefox provides a stable CSSPseudoElement implementation, so I would just suppress this type.